web/src/locales/en-US/component.ts

```diff
@@ -25,12 +25,12 @@ export default {
   'component.global.document': 'Document',
   'component.global.enable': 'Enable',
   'component.global.scope': 'Scope',
-  'component.global.data.editor': 'Data Editor',
+  'component.global.data.editor': 'CodeMirror',
   'component.global.delete': 'Delete',
   'component.global.cancel': 'Cancel',
   'component.global.submit': 'Submit',
   'component.global.create': 'Create',
-  'component.global.createWithEditor': 'Create with Editor',
```

Comment:
I think `Create with Editor` is easier to understand, those who have not used `CodeMirror ` before may not know what codemirror does. 🤔

Reply:
Not sure, Create with Editor is more like a description than a feature name IMO :)

How about other changes?